### ISSUE TITLE
Wrong variables types rendering by msvc lldb with kotlin plugin 

### ISSUE THREAD

**Issue by Undin** [intellij-rust/intellij-rust#6922]
## Environment

* **IntelliJ Rust plugin version:** 0.3.142.3705-211
* **Kotlin plugin version:** 211-1.4.21-release-IJ6222.3
* **Rust toolchain version:** 1.50.0 (cb75ad5db 2021-02-10) x86_64-pc-windows-msvc
* **IDE name and version:** IntelliJ IDEA 2021.1 EAP Ultimate Edition (IU-211.6222.4)
* **Operating system:** Windows 10 10.0
* **Macro expansion engine:** new
* **Name resolution engine:** old

## Problem description
If you try to debug rust application in IntelliJ IDEA Ultimate with enabled kotlin plugin, all types are rendered as `ObjHeader *`. If kotlin plugin is disabled, everything is fine

| With Kotlin plugin | Without Kotlin plugin |
|-|-|
|![msvc-debugger-types-rendering-with-kotlin](https://user-images.githubusercontent.com/2539310/110022898-8c953d80-7d3d-11eb-91ad-92a225b81a0c.PNG) | ![msvc-debugger-types-rendering-without-kotlin](https://user-images.githubusercontent.com/2539310/110023469-47bdd680-7d3e-11eb-8e4c-5fe0b3afe7c8.PNG) |

Not sure that we should fix in Rust plugin, most likely it's Kotlin plugin issue

## Steps to reproduce
```rust
fn main() {
    let buf = String::from("askldnsajdnsaj");
    let x = 123;
    let vec = vec![1, 2, 3, 4];
    foo(&vec);
    println!("{} {} {:?}", x, buf, vec);
}

fn foo(x: &[i32]) {}
```

**Comment by Undin** [comment]
The issue in kotlin tracker: https://youtrack.jetbrains.com/issue/KTIJ-3776

**Comment by Undin** [comment]
Fixed in latest IntelliJ IDEA 2021.2 EAP (212.3724.25)
